### PR TITLE
Team Codeowners and Phoenix has more files

### PR DESCRIPTION
**Pull request by jyejare**
### Problem Statement
- Teams are now T1 Reviewers and hence they should have notification for the file changes made in their own team test file.
- Tier1 Reviewer are optional and hence are not going to be getting notification.
- Phoenix team looks to have more files.

### Solution
- Codeowners file is updated with per test file ownership per team.
- Team Phoenix files are updated.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -21,70 +19,337 @@
...
+# Rocket
+
+tests/foreman/api/test_ansible.py @SatelliteQE/team-rocket
+tests/foreman/api/test_classparameters.py @SatelliteQE/team-rocket
+tests/foreman/api/test_computeprofile.py @SatelliteQE/team-rocket
+tests/foreman/api/test_computeresource_azurerm.py @SatelliteQE/team-rocket
+tests/foreman/api/test_computeresource_gce.py @SatelliteQE/team-rocket
+tests/foreman/api/test_computeresource_libvirt.py @SatelliteQE/team-rocket
+tests/foreman/api/test_computeresource_vmware.py @SatelliteQE/team-rocket
+tests/foreman/api/test_contentview.py @SatelliteQE/team-rocket
```
Comment (shweta83) [inline .github/CODEOWNERS:151]: This component is owned by phoenix team.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -21,70 +19,337 @@
...
+tests/foreman/api/test_convert2rhel.py @SatelliteQE/team-rocket
+tests/foreman/api/test_discoveredhost.py @SatelliteQE/team-rocket
+tests/foreman/api/test_discoveryrule.py @SatelliteQE/team-rocket
+tests/foreman/api/test_environment.py @SatelliteQE/team-rocket
+tests/foreman/api/test_host.py @SatelliteQE/team-rocket
```
Comment (shweta83) [inline .github/CODEOWNERS:156]: This component is owned by endeavour team.

File: .github/CODEOWNERS (hunk 1 above)
Comment (shweta83) [reply .github/CODEOWNERS:151]: I see there are few tests for Rocket team.

File: .github/CODEOWNERS (hunk 2 above)
Comment (shweta83) [reply .github/CODEOWNERS:156]: I see there are few tests for Rocket team.

**Review by shweta83 — APPROVED** [review]
ACK for Rocket team components.

**Review by Griffin-Sullivan — APPROVED** [review]

**Comment by jyejare** [comment]
@vsedmik @pondrejk Please review.

**Review by vsedmik — APPROVED** [review]
Looks good to me, just one note - should we keep the original `/tests/foreman/virtwho/ @SatelliteQE/phoenix`, since _everything_ in that folder falls under Phoenix?

**Comment by jyejare** [comment]
@vsedmik Thanks for the review, changes are now updated.

**Review by pondrejk — APPROVED** [review]

**Comment by jyejare** [comment]
All 4 ACKs from QEPOCs lets me merge this to production now.

**Pull request merged**